build/_k-test.shade

```diff
@@ -58,7 +59,10 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                Dnx("test" + testArgs, projectFolder);
+                if (!IsLinux || string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO))
```

Comment:
This is getting crazy. Just check for `1` and `true` (lowercase as far as I'm concerned) and ignore everything else.